### PR TITLE
[monorepo] feat: Auto-merge of dependabot prs

### PR DESCRIPTION
**Pull request by Wayonb**
## What is the current behavior?
Dependabot PRs are manually merge into GitHub

## What's the issue?
Merge of PR is manual

## How have you changed the behavior?
Add a GitHub action which will auto merge all PRs from dependabot after the CI checks pass.

Note: commit-message prefix field is limited to 15 characters, monorepo was used in such case.

## How was this change tested?
Tested the auto merge of Dependabot PR

### CODE REVIEW

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@ updates:
       day: sunday
     target-branch: dev
     labels: [catbuffer-parser]
+    commit-message:
+      prefix: '[monorepo] fix'
```
Comment (Jaguar0625) [inline .github/dependabot.yaml:12]: 1. why monorepo
2. will this pass commit message linter ?

File: .github/dependabot.yaml [hunk 2]
```diff
@@ -66,6 +80,8 @@ updates:
       day: sunday
     target-branch: dev
     labels: [sdk-javascript]
+    commit-message:
+      prefix: '[sdk] fix'
```
Comment (Jaguar0625) [inline .github/dependabot.yaml:84]: sdk/javascript ?

File: .github/workflows/auto-merge.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+name: Dependabot auto-approve-merge
+
+# yamllint disable-line rule:truthy
```
Comment (Jaguar0625) [inline .github/workflows/auto-merge.yaml:3]: can you quote value instead?

File: .github/dependabot.yaml (hunk 1 above)
Comment (Wayonb) [reply .github/dependabot.yaml:12]: 1. I noted above that the commit-message prefix field is limited to 15 characters.  ``[catbuffer/parser] fix`` is too long.  Do we want to create different tag?
2. It should create a commit title of ``[monorepo] fix: xxxxxxx``` where xxxxxxx is the original commit title.

File: .github/dependabot.yaml (hunk 2 above)
Comment (Wayonb) [reply .github/dependabot.yaml:84]: Same limitation on prefix field

File: .github/workflows/auto-merge.yaml (hunk 3 above)
Comment (Wayonb) [reply .github/workflows/auto-merge.yaml:3]: hmm....not sure if you can quote the ``on`` but I will try

File: .github/dependabot.yaml (hunk 1 above)
Comment (Jaguar0625) [reply .github/dependabot.yaml:12]: i would drop fix ?

File: .github/workflows/auto-merge.yaml (hunk 3 above)
Comment (Wayonb) [reply .github/workflows/auto-merge.yaml]: cool.  Added quotes worked

File: .github/dependabot.yaml (hunk 1 above)
Comment (Wayonb) [reply .github/dependabot.yaml:12]: Let me see which ones will fit.

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**